### DISCUSSION TITLE
Default Styling

### DISCUSSION THREAD

**Discussion by rmorshea** [idom-team/idom#889] [Ideas]
Right now, IDOM doesn't come with any styled elements out of the box. This basically means you need to bring your own which, if you just want to get something working quickly, is a bit of a pain. Perhaps we could include some default styling to make this easier.

As it happens, I just discovered [Pico.css](https://picocss.com/) which seems like a great solution to this problem. In short, it applies styling to HTML elements without a class. This means users can get a decent looking interface without any extra effort. Here's a preview of what Pico.css can do:

https://user-images.githubusercontent.com/23470684/126863110-94061cf1-36ea-4697-94bd-2e1071a95a2f.mp4

We could either turn this styling on by default, have an option to turn on default styling when you configure your backend, or do the latter but turn on default styling in `idom.run`. I think I'm partial to having the default styling on by default because I think being able to create good looking interfaces with little effort will leave a better impression - I'd rather have a user come back asking how to turn this off than to look at their UI, think it's ugly, and then leave without learning that you can turn on a default style.

**Comment by Archmonger** [comment]
It would be convenient to be enabled by default. But ReactJS is ugly by default as well, so that might be an odd mismatch.

Also may cause unexpected consequences once users start frequently using the JavaScript components API and/or `idom-build`.
